src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -74,11 +74,14 @@ $pf-v6--include-toolbar-breakpoints: true !default;
   flex-wrap: wrap;
 }
 
+.#{$toolbar}__content {
+  @include pf-v6-hidden-visible(grid);
+}
+
 // - Toolbar - Toolbar content
 .#{$toolbar},
 .#{$toolbar}__content {
   position: relative;
-  display: grid;
```

Comment:
Since this removes `display: grid;` from `.#{$toolbar}`, too, we'll need to add that back. You can probably just add it as the first `.#{$toolbar}` style here

https://github.com/patternfly/patternfly/blob/c7a6d8b0234a64c478bbb49578847d0511031dca/src/patternfly/components/Toolbar/toolbar.scss#L88-L89